client/src/Components/Redirect.js

```diff
@@ -23,11 +22,7 @@ function Redirect({ isAdminOnly }) {
     }
   }, [history, user, isAdminOnly]);
 
-  return (
-    <div className="padding-9">
-      <Spinner />
-    </div>
-  );
+  return <></>;
```

Comment:
I think this can just end without a return and be ignored when rendering.  Or `return null` to explicitly not render anything.

Reply:
I'll change it to return null... React 16 still throws an error if undefined is returned (changed in React 18 I believe)

Reply:
Ah, you're right.  It threw before 18.